generic_config_updater/generic_config_updater.conf.json

```diff
@@ -58,19 +73,19 @@
             "validate_commands": [ ]
         },
         "rsyslog": {
-            "validate_commands": [ "generic_config_updater.services_validator.rsyslog_validator" ]
```

Comment:
`validate_commands` is used for post health check after ConfigDB modification. What you want to achieve is to check the modification before it is applied to ConfigDB.

Reply:
I created a new conf file specific to validators that need to be run before the change is applied to ConfigDB

Reply:
Could you keep original name `services_validator`? there is nothing wrong with old code.

Reply:
I added the new validator `is_mellanox_device_validator` to the originally named `services_validator.py` file, so I changed to `validator.py` 

If `is_mellanox_device_validator` would be better placed somewhere else, I will move it and leave the original file name `services_validator.py`

Reply:
You can place new validator in `generic_config_updater.move_validator`, and still keep old code.

Reply:
This new validator is not a MoveValidator, which validates every intermediary move. The preexisting MoveValidators come in during patch sorting.  For MoveValidators, if validation fails, the PatchSorter algorithm will continue searching for a new valid move. For this use case, we don't need to start patch sorting at all.

Please see a comment I made on a previous PR https://github.com/sonic-net/sonic-utilities/pull/2545 describing why we are adding this new infra for one-shot validation at the start (as opposed to MoveValidators for every intermediary move).

We don't want to validate each intermediate move to get to the final state, as is done by MoveValidators- we just need to validate the initial request. MoveValidators continue searching for a different move that is 'valid' if it finds one invalid - possibly by deleting the parent and then readding other children. This is not the behavior we desire. We only need a check of the initial request (reflected in Json diff).

Reply:
Could you keep original name services_validator? there is nothing wrong with old code.

Reply:
Yes, I changed it back to services_validator. I initially changed it away because I added a non-service validator to this file, but this new non-service validator is now in a new file `field_operation_validators.py`